client/my-sites/patterns/pages/category/style.scss

```
@@ -0,0 +1,98 @@
+@import "@wordpress/base-styles/breakpoints.scss";
+@import "@automattic/components/src/styles/typography";
+
+.category-navigation {
```

```suggestion
.is-section-patterns .category-navigation {
```
We should qualify the `.category-navigation` selector somehow.